pages/k8s/charm-kubernetes-worker.md

```diff
@@ -352,21 +352,278 @@ update the configuration for the workers:
     ```
 
 ## Actions
+<!-- ACTIONS STARTS -->
+<!-- AUTOGENERATED TEXT - DO NOT EDIT -->
 
-The kubernetes-worker charm models a few one time operations called
-[Juju actions](https://jaas.ai/docs/working-with-actions) that can be run by
-Juju users.
+The following actions are available. You can run an action with the following
 
-### Pause
-
-Pausing the workload enables administrators to both drain and cordon
-a unit for maintenance.
-
-### Resume
-
-Resuming the workload will uncordon a paused unit. Workloads will automatically
-migrate unless otherwise directed via their application declaration.
+```bash
+juju run-action kubernetes-worker ACTION [parameters] [--wait]
+```
 
+<div class="row">
+  <div class="col-2">
+     <h5> cis-benchmark</h5>
+  </div>
+  <div class="col-7">
+    <p>Run the CIS Kubernetes Benchmark against snap-based components.
+</p>
+  </div>
+</div>
+
+<div class="row">
+  <div class="col-2">
+  </div>
+  <div class="col-7">
+    <p>This action has the following parameters:</p>
+    <table class ="u-table-layout--auto">
+      <thead>
+        <tr>
+          <th>parameter</th>
+          <th>Description</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td><pre>apply</pre></td>
+          <td><p>Apply remediations to address benchmark failures. The default, 'none', will not attempt to fix any reported failures. Set to 'conservative' to resolve simple failures. Set to 'dangerous' to attempt to resolve all failures. Note: Applying any remediation may result in an unusable cluster.</p>
+          <p><strong>Default:</strong> none</p>
+        </td>
+        </tr>
+        <tr>
+          <td><pre>config</pre></td>
+          <td><p>Archive containing configuration files to use when running kube-bench. The default value is known to be compatible with snap components. When using a custom URL, append '#<hash_type>=<checksum>' to verify the
+archive integrity when downloaded. <strong>Default:</strong> https://github.com/charmed-kubernetes/kube-bench-config/archive/cis-1.5.zip#sha1=cb8e78712ee5bfeab87d0ed7c139a83e88915530</p>
```

Comment:
This bit seems to break rendering somehow.  I think it's because of the `'#<hash_type>=<checksum>'` bit, which probably needs to be changed to `'#&lt;hash_type&gt;=&lt;checksum&gt;'`

Reply:
the rendering is broken because of various issues with the markdown parser not knowing where the HTML starts and ends. i need to do some more thorough sanitising of the text that goes in the table, that's what breaks it (see the other entries are fine)

Reply:
though that massive unbroken string does cause other issues. Maybe I will break it with a &#9166; character. But then I have to guess where. That's always a problem with this stuff - it is difficult to pre-process stuff without knowing how it renders - is this string going to fit in a box? How many lines in a table will this fill? etc

Reply:
So, I fixed up the html so the markdown parser could differentiate and it solved that problem. It's painful because the templating doesn't allow for spacing, spacing is how the parser knows when the HTML is over. 

Anyhow, I *still* need to sanitise the explanatory test, but at least it is in the right place now ;)